changelog/unreleased/issue-2951

```diff
@@ -0,0 +1,9 @@
+Bugfix: restic generate, help and self-update no longer check passwords
+
+The commands restic generate, restic help and restic self-update don't need
```

Comment:
I think we don't need to repeat `restic` here multiple times. I'd also enclose the command names in backticks \` to highlight them in the changelog.

Reply:
Done.